Sources/Player/PlaybackEngine/Internal/Metrics/Metrics.swift

```diff
@@ -158,6 +158,7 @@ private extension Metrics {
 		guard let idealStartTime else {
 			// idealStartTime isn't set, so there has been no intention to start playback.
 			// If no one has tried to play, playback cannot fail.
+			PlayerWorld.logger?.log(loggable: PlayerLoggable.metricsNoIdealStartTime)
```

Comment:
We are already logging this in our StreamingMetrics, no? Do we gain anything by adding this here?

Reply:
Yes, it is in the end message; however I think this could help me understanding the whole flow (in case this log is related to any issue we currently have) since currently we have them in different tools.